plugins/pulp_rpm/plugins/migrations/0025_export_relative_url.py

```diff
@@ -0,0 +1,62 @@
+"""
+Migration to normalize relative_url, and populate relative_url on
+export distributors
+"""
+from pulp.server.db import connection
+
+
+def _find_yum_distributor(distributors):
+    for distributor in distributors:
+        if distributor['distributor_type_id'] == 'yum_distributor':
+            if 'config' in distributor and 'relative_url' in distributor['config']:
+                return distributor
+    return None
+
+
+def _clean_relative_url(relative_url):
+    if relative_url.startswith('/', 0, 1):
+        return relative_url[1:]
+    else:
+        return relative_url
```

Comment:
Also, it's *hilarious* that we were making/accepting absolute relative URLs at some point. :D